client/src/components/History/Content/Collection/JobStateSummary.js

```diff
@@ -6,8 +6,8 @@
  */
 import { STATES } from "../model/states";
 
-const NON_TERMINAL_STATES = [STATES.NEW, STATES.WAITING, STATES.QUEUED, STATES.RUNNING];
-const ERROR_STATES = [STATES.ERROR, STATES.DELETED];
+const NON_TERMINAL_STATES = ["new", "waiting", "queued", "running"];
```

Comment:
This is not complete, so will break. You can find the source of truth for **Job** states in https://github.com/mvdbeek/galaxy/blob/c0fc0a853edb102a685f1343efc04dace4b044aa/lib/galaxy/model/__init__.py#L1049-L1075

Reply:
@mvdbeek I added those additional non-terminal states to this list. 

I also added the deleted state -- I'm still not sure if deleted or discarded is the deleted state or if both are possilble. 
`states.js` only listed discarded. Anyways, I've included both for now. Let me know if I should remove one.

Reply:
discarded is a dataset state, that can never occur here and we should drop it